adapters/dmx/dmx_test.go

```diff
@@ -0,0 +1,449 @@
+package dmx
```

Comment:
In the future, you should look at the json test machinery that has been developed for adapters. It does improve test maintainability and readability. But may not be worth the time of converting all these hard coded tests over. But do look at creating`dmxtest/params/race/banner.json`, this would allow us to include your adapter in the race condition tests.